Assets/Scripts/Audio/TimeStamper.cs

```diff
@@ -0,0 +1,456 @@
+                    newDirection = _timeStampTweakAmount * Input.mouseScrollDelta.y * Vector2.right;
+                else
+                {
+                    if (Input.GetKey(_decreaseTimeStampKey)) //decrease
+                        newDirection = Vector2.left * _timeStampTweakAmount;
+
+                    if (Input.GetKey(_increaseTimeStampKey)) //increase
+                        newDirection = Vector2.right * _timeStampTweakAmount;
+                }
+
+                _currentSelectedTimeStamp.lineData.startLinePoint += newDirection;
+                _currentSelectedTimeStamp.lineData.endLinePoint += newDirection;
+
+                //update song time of the current selected time stamp
+                _currentSelectedTimeStamp.songTime =
+                    _waveformDrawer.CalculateSongTimeBasedOnPosition(_currentSelectedTimeStamp.lineData.startLinePoint);
+            }
+        }
+
+        /// <summary>
+        /// Returns the closest time stamp based on input origin position.
+        /// </summary>
+        /// <param name="originPosition">Origin of the distance check.</param>
+        /// <returns>Data of the closest time stamp.</returns>
+        private TimeStampData GetClosestTimeStamp(Vector2 originPosition)
+        {
+            TimeStampData bestTarget = default;
+            float closestDistanceSqr = Mathf.Infinity;
+
+            foreach (TimeStampData timeStampData in _timeStamps)
+            {
+                Vector3 directionToTarget = timeStampData.lineData.startLinePoint - originPosition; //get the direction to the target
+                float dSqrToTarget = directionToTarget.sqrMagnitude; //get the squared distance to the target
+                
+                if (dSqrToTarget < closestDistanceSqr) //check if the current time stamp is closer than the previous closest time stamp
+                {
+                    closestDistanceSqr = dSqrToTarget;
+                    bestTarget = timeStampData;
+                }
+            }
+
+            return bestTarget;
+        }
+
+        /// <summary>
+        /// Method for showing the keybinds in the console.
+        /// </summary>
+        private void HandleShowKeybinds()
+        {
+            Debug.Log("Place TimeStamp Key: " + _placeTimeStampKey);
+            Debug.Log("Undo TimeStamp Key: CTRL + " + _undoTimeStampKey);
+            Debug.Log(" "); //empty line
+
+            Debug.Log("Select TimeStamp Key: CTRL");
+            Debug.Log("Delete selected TimeStamp Key: CTRL + " + _deleteCurrentTimeStampKey);
+            Debug.Log(" "); //empty line
+
+            Debug.Log("Export TimeStamps Key: CTRL + " + _exportTimeStampsKey);
+            Debug.Log(" "); //empty line
+
+            Debug.Log("Increase TimeStamp Key: CTRL + " + _increaseTimeStampKey);
+            Debug.Log("Decrease TimeStamp Key: CTRL + " + _decreaseTimeStampKey);
+            Debug.Log(" "); //empty line
+        }
```

Comment:
Make this an optional by adding an if statement.

Reply:
This method gets called when the keybinds button is pressed (in custom editor window) so it is already optional